playbooks/installer/ospd/virthost.yml

```diff
@@ -116,5 +116,7 @@
         lineinfile:
             dest: "/home/{{ installer.user.name }}/.ssh/authorized_keys"
             line: "{{ undercloud_public_key.stdout }}"
+            create: yes
```

Comment:
I would recommend to use the athorized_keys module instead of lineinfile here: 

```
       authorized_key:
           user: "{{ installer.user.name }}"
           key: "{{ undercloud_public_key.stdout }}"
```

This module will create file if it is not present